watson-developer-cloud/discovery-components
# chore: add unit test for CIDocument details utility

#### What do these changes do/fix?

Adds unit tests to the CIDocument details utility for higher test coverage of our code.

#### How do you test/verify these changes?

`yarn test details` and see that details util tests pass.

Run `yarn test:coverage` and view the function and branch coverage of the CIDocument\utils\details.ts file as see the results are above desired threshold 80% - branch, 100% - function.
